app/templates/macros/nav_macros.html

```diff
@@ -17,19 +17,36 @@
     {% endif %}
 {% endmacro %}
 
-{% macro account_items(current_user) %}
+{% macro action_items(current_user) %}
     {% if current_user.is_authenticated() %}
-        <a href="{{ url_for('account.manage') }}" class="item">Your Account</a>
-        <a href="{{ url_for('account.logout') }}" class="item">Log out</a>
-    {% else %}
-        <a href="{{ url_for('account.register') }}" class="item">Register</a>
-        <a href="{{ url_for('account.login') }}" class="item">Log in</a>
+        <div class="item ui mini search">
```

Comment:
so html indentation is usually 2 spaces, but somehow we started doing 4 spaces so let's make it consistent and have each indent be 4 spaces please.